add_npm_package.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+# Run as ./add_npm_package.sh name version strategy
+program_exists() {
+  which "$@" &> /dev/null;
+}
+
+if !(program_exists crudini); then
+  echo "crudini is not installed - installing..."
+  sudo yum install crudini
+  echo "crudini installed..."
+fi
+
+if !(program_exists npm2rpm); then
+  echo "npm2rpm is not installed - installing..."
+  sudo npm install -g npm2rpm
+  echo "npm2rpm installed..."
+fi
+
+echo -n "Making directory..."
+mkdir nodejs-$1
+echo "FINISHED"
+echo -n "Creating specs and downloading sources..."
+npm2rpm -n $1 -v $2 -s $3
+echo "FINISHED"
+echo -n "Copying specs..."
+cp npm2rpm/SPECS/* nodejs-$1
+echo "FINISHED"
+echo -n "Copying sources..."
+cp npm2rpm/SOURCES/* nodejs-$1
+echo "FINISHED"
+rm -r npm2rpm
+echo -n "Setting tito props..."
+# Get tito.props whitelists and add node package
+el7whitelist=$(crudini --get rel-eng/tito.props foreman-nightly-nonscl-rhel7 whitelist)
+el7whitelist=$(echo "$el7whitelist nodejs-$1" | tr " " "\n" | sort)
```

Comment:
set LC_COLLATE when sorting as different people's locales will change the list order (e.g. comps_doc.sh uses en_GB)